templates/islandora-solr-metadata-display.tpl.php

```diff
@@ -30,6 +30,14 @@
           </dd>
           <?php $row_field++; ?>
         <?php endforeach; ?>
+        <?php if (!empty($object_views_count_view)): ?>
+          <dt>
+            Object views
```

Comment:
was thinking that at the time, but didn't figure it would really matter in this particular case. want me to update the pr?